app/models/setting/email.rb

```diff
@@ -0,0 +1,81 @@
+require 'facter'
+
+class Setting::Email < Setting
+  NON_EMAIL_YAML_SETTINGS = %w(send_welcome_email email_reply_address email_subject_prefix)
+
+  def self.load_defaults
+    Foreman::Deprecation.deprecation_warning('1.15', "'email.yaml' file configuration has been deprecated, email configuration has been moved to Foreman settings") if mailconfig.present?
+    # Check the table exists
+    return unless super
+    domain = Facter.value(:domain) || SETTINGS[:domain]
+    email_reply_address = "Foreman-noreply@#{domain}"
+
+    self.transaction do
+      [
+        self.set('email_reply_address', N_("Email reply address for emails that Foreman is sending"), email_reply_address, N_('Email reply address')),
+        self.set('email_subject_prefix', N_("Prefix to add to all outgoing email"), '[foreman]', N_('Email subject prefix')),
+        self.set('send_welcome_email', N_("Send a welcome email including username and URL to new users"), false, N_('Send welcome email')),
+        self.set('delivery_method', N_("Method used to deliver email"), 'sendmail', N_('Delivery method'), nil, { :collection => Proc.new {{:sendmail => _("Sendmail"), :smtp => _("SMTP")}}}),
+        self.set('smtp_enable_starttls_auto', N_("SMTP automatic STARTTLS"), true, N_('SMTP enable StartTLS auto')),
+        self.set('smtp_openssl_verify_mode', N_("When using TLS, you can set how OpenSSL checks the certificate"), '', N_('SMTP OpenSSL verify mode'), nil, { :collection => Proc.new {{'' => _("no OpenSSL certificate verifier"), :none => _("none"), :peer => _("peer"), :client_once => _("client_once"), :fail_if_no_peer_cert => _("fail_if_no_peer_cert")}}}),
```

Comment:
1. "no OpenSSL certificate verifier" should probably be "Default verification mode", which looks like "peer". Verification isn't disabled by default. I would probably keep it as a separate value to peer though, that's set inside OpenSSL.
1. "fail_if_no_peer_cert" etc are not good English strings to translate, either use proper descriptions or don't translate them.